core/src/main/java/com/adobe/aio/exception/feign/RetryAfterError.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package com.adobe.aio.exception.feign;
+
+import static java.time.format.DateTimeFormatter.RFC_1123_DATE_TIME;
+
+import feign.FeignException;
+import feign.Response;
+import java.time.Instant;
+import java.time.format.DateTimeParseException;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RetryAfterError extends FeignException {
```

Comment:
actually @shikhartanwar we should use the `RetryableException` instead here
to leverage the Feign retryer see https://github.com/adobe/aio-lib-java/pull/151